ml-agents/mlagents/trainers/ppo/optimizer_tf.py

```diff
@@ -338,9 +338,18 @@ def _construct_feed_dict(
         if self.policy.output_pre is not None and "actions_pre" in mini_batch:
             feed_dict[self.policy.output_pre] = mini_batch["actions_pre"]
         else:
-            feed_dict[self.policy.output] = mini_batch["actions"]
-            if self.policy.use_recurrent:
-                feed_dict[self.policy.prev_action] = mini_batch["prev_action"]
+            if self.policy.use_continuous_act:  # For hybrid action buffer support
```

Comment:
Should that comment be a TODO?

Reply:
Not a TODO, just pointing out the fact that im getting continuous/discrete explicitly due to the buffer changes to support hybrid actions

Reply:
Ah, forgot this was TF code, so the implicit TODO is "burn it all"